src/hdtSkyrimPhysicsWorld.cpp

```
@@ -46,7 +46,7 @@ namespace hdt
 		m_accumulatedInterval = 0;
 	}
 
-	SkyrimPhysicsWorld::~SkyrimPhysicsWorld(void)
+	SkyrimPhysicsWorld::~SkyrimPhysicsWorld(void) noexcept
```

Clang (14.0.6)

[warning] use '= default' to define a trivial destructor

(modernize-use-equals-default)